recipes-soletta/soletta/soletta-ptest.inc

```diff
@@ -0,0 +1,18 @@
+inherit ptest
+
+SRC_URI += "file://run-ptest \
+           "
+
+do_compile_ptest() {
+        oe_runmake CFLAGS="--sysroot=${STAGING_DIR_TARGET} -pthread -lpcre" TARGETCC="${CC}" TARGETAR="${AR}" "tests"
```

Comment:
@barbieri 
By double check, there is no need to add CFLAGS when build test binaries. 
I will amend the patch. Thanks.